3DAmsterdam/Assets/Amsterdam3D/Scripts/SelectionTools/BoxSelect.cs

```diff
@@ -1,39 +1,35 @@
-﻿using System.Collections;
-using System.Collections.Generic;
-using UnityEngine;
-using UnityEngine.UI;
-using UnityEngine.Networking;
-using Amsterdam3D.CameraMotion;
-using System.Linq;
-using Assets.Amsterdam3D.Scripts.Camera;
+﻿using UnityEngine;
 using Amsterdam3D.CameraMotion;
-using UnityEngine.Events;
-
+using System.Collections.Generic;
 
-namespace Amsterdam3D.SelectionTools
+namespace Amsterdam3D.Interface
 {
-
     public class BoxSelect : SelectionTool
     {
-        [SerializeField]
+		private const float requiredPixelDragDistance = 10.0f;
+		[SerializeField]
         private GameObject selectionBoxPrefab;
+
         private RectTransform selectionBox;
 
```

Comment:
Misschien de private waardes assigne met = default; als ze geen assignment hebben zodat er geen inspector warnings komen?

Reply:
yess, zal dat in vervolg proberen te blijven doen 👍